proto/frequenz/api/microgrid/v1/microgrid.proto

```diff
@@ -130,6 +130,37 @@ service Microgrid {
     };
   }
 
+  // Adds inclusion bounds for a given metric of a given component, and returns
+  // the UTC timestamp until which the given inclusion bounds will stay in
+  // effect.
+  //
+  // Inclusion bounds refer to the range of values that are allowed for the
+  // metric. If these bounds for a metric are [[`lower_1`, `upper_1`],
+  // [`lower_2`, `upper_2`]], then
+  // this metric's `value` needs to comply with the constraint
+  // `lower_1 <= value <= upper_1` AND `lower_2 <= value <= upper_2`.
+  //
+  // If multiple inclusion bounds have been provided for a metric, then the
+  // overlapping bounds are merged into a single bound, and non-overlapping
+  // bounds are kept disjointed.
+  // E.g. if the bounds are [[0, 10], [5, 15], [20, 30]], then the resulting
+  // bounds will be [[0, 15], [20, 30]].
+  //
+  // The following diagram illustrates how bounds are applied:
+  // ```
+  //   lower_1  upper_1
+  // <----|========|--------|========|-------->
+  //                     lower_2  upper_2
+  // ```
```

Comment:
this contradicts the above..."needs to comply with the constraint
  // `lower_1 <= value <= upper_1` AND `lower_2 <= value <= upper_2`". If the statement is true then no value should be allowed in this example.

Reply:
Good catch! Fixed